core/domain/subtopic_page_services.py

```diff
@@ -341,6 +342,35 @@ def delete_subtopic_page(
         topic_id, subtopic_id)
 
 
+def get_model_instances_for_delete_subtopic_page(
```

Comment:
Please name your functions so that they describe what the function is doing exactly, not what the results of the function are being used for.